### PR TITLE
[RHACS] ROX16236 Integrating with image vulnerability scanners documentation

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
3.73, 3.74, 4.0

Issue:
https://issues.redhat.com/browse/ROX-16236

Link to docs preview:
https://59187--docspreview.netlify.app/openshift-acs/latest/integration/integrate-with-image-vulnerability-scanners.html

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs` and cherry-pick into:
    - `rhacs-docs-4.0`
    - `rhacs-docs-3.74`
    - `rhacs-docs-3.73`
- Google doc: https://docs.google.com/document/d/1Bv7VmJnVVJEXd6YNYkzsvWo5RrMvkJ2JYKQF0BOBVgY/edit

### CODE REVIEW

**Comment by mfosterrox** [comment]
@agantony LGTM! Thanks.

File: integration/integrate-with-image-vulnerability-scanners.adoc
Comment (kcarmichael08) [inline integration/integrate-with-image-vulnerability-scanners.adoc:1]: Need to add :_content-type: ASSEMBLY to header per [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata)

File: integration/integrate-with-image-vulnerability-scanners.adoc [hunk 1]
```diff
@@ -6,13 +6,40 @@ include::modules/common-attributes.adoc[]
...
+* Generic Docker registries (any generic Docker or Open Container Initiative-compliant image registries, for example, DockerHub, `gcr.io`, `mcr.microsoft.com`)
+* Google Container Registry
+* Google Artifact Registry
+* IBM Cloud Container Registry
+* JFrog Artifactory
+* Microsoft Azure Container Registry (ACR)
+* Red Hat Quay
+* Red Hat registry (`registry.redhat.io`, `registry.access.redhat.com`)
+
+This enhanced support gives you greater flexibility and choice to manage your container images in your preferred registry.
```
Comment (kcarmichael08) [inline integration/integrate-with-image-vulnerability-scanners.adoc:26]: suggestion to change "to manage" to "in managing" but your call

File: integration/integrate-with-image-vulnerability-scanners.adoc [hunk 2]
```diff
@@ -6,13 +6,40 @@ include::modules/common-attributes.adoc[]
...
+You can set up {product-title-short} to obtain image vulnerability data from the following commercial container image vulnerability scanners:
+
+* {product-title-short} Scanner (recommended)
 * link:https://github.com/quay/clair[Clair]
 * link:https://cloud.google.com/container-registry/docs/container-analysis[Google Container Analysis]
 * link:https://quay.io[Red Hat Quay]
 
+[IMPORTANT]
+====
+{product-title-short} Scanner is the most recommended and preferred among all other commercial container image vulnerability scanners. Read more about scanning container images with {product-title-short} Scanner in xref:../operating/examine-images-for-vulnerabilities.adoc#scanning-images_examine-images-for-vulnerabilities[Scanning images].
```
Comment (kcarmichael08) [inline integration/integrate-with-image-vulnerability-scanners.adoc:40]:
```suggestion
{product-title-short} Scanner is the preferred image vulnerability scanner to use with {product-title-short}. For more information about scanning container images with {product-title-short} Scanner, see xref:../operating/examine-images-for-vulnerabilities.adoc#scanning-images_examine-images-for-vulnerabilities[Scanning images].
```

Comment (kcarmichael08) [reply integration/integrate-with-image-vulnerability-scanners.adoc:40]: OpenShift documentation is pretty standard in its references ("for more information, see x") so I've made that suggestion here. Also I suggested a change to the first sentence to kind of focus in on the suggestion that our scanner works best with our product (even though you can use other scanners); WDYT?

**Review by kcarmichael08 — COMMENTED** [review]
Nice job of getting what info needs to be updated and clarifying! Couple of suggestions; otherwise LGTM.

File: integration/integrate-with-image-vulnerability-scanners.adoc (hunk 2 above)
Comment (agantony) [reply integration/integrate-with-image-vulnerability-scanners.adoc:40]: SGTM!

**Review by RTann — APPROVED** [review]
LGTM from ACS eng

**Review by RTann — CHANGES_REQUESTED** [review]
I lied I just noticed Sonatype Nexus is missing from the registry list

**Comment by agantony** [comment]
> I lied I just noticed Sonatype Nexus is missing from the registry list

@RTann Added :+1: 
PS: https://github.com/openshift/openshift-docs/pull/59187/commits/d2d2388c1f79561c06245fa4457b286c0521cb1c

**Comment by bscott-rh** [comment]
@agantony the auto-cherrypick to rhacs-docs-3.73 failed. Ping me here or on slack when you have a manual cherrypick PR up.

**Comment by agantony** [comment]
> @agantony the auto-cherrypick to rhacs-docs-3.73 failed. Ping me here or on slack when you have a manual cherrypick PR up.

@bscott-rh Manual CP PR: https://github.com/openshift/openshift-docs/pull/59447

**Pull request merged**